docs/upgrade/focal_migration.rst

```diff
@@ -0,0 +1,240 @@
+preserve their existing *Source* and *Journalist Interface* onion URLs.
+
+.. note::
+   Coordinate with journalists and sources during this migration. Your
+   instance will be offline while you perform these steps.
+
+   Coordinate with the team maintaining your Landing Page. You may wish to
+   publish a notice about a planned maintenance window. Depending on
+   :ref:`your migration scenario <choose_migration_path>`, you may also
+   need to coordinate the publication of your new *Source Interface* onion 
+   URL so that sources can reach you.
+
+
+Preparation
+~~~~~~~~~~~
+Before migrating, complete the following steps:
+
+#. :ref:`Consider a hardware upgrade <consider_hardware_upgrade>`
+#. :ref:`Choose your migration path <choose_migration_path>` and plan your
+   maintenance window
+#. :ref:`Check your SecureDrop version (servers) <check_server_versions>`
+#. :ref:`Check your SecureDrop version (workstations) <check_workstation_versions>`
+#. :ref:`Verify SSH access <verify_ssh_access>`
+
+
+.. _consider_hardware_upgrade:
+
+Consider a hardware upgrade
+---------------------------
+
+If you are running hardware that is not currently listed in our
+:ref:`hardware recommendations <Specific Hardware Recommendations>`, we
+recommend that you also plan a hardware refresh as part of this migration,
+particularly if your hardware is reaching end-of-life.
+This has the following benefits:
+
+   - It ensures that all system components will continue to receive security
+     updates.
+   - It reduces the risk of hardware compatibility issues with future
+     releases of SecureDrop.
+   - It will allow you to keep your current installation online during much of
+     the two-day maintenance window.
+   - If your hardware is due for replacement anyway, combining the OS upgrade 
+     and the hardware upgrade will save you time. 
+
+If you have a support agreement with Freedom of the Press Foundation,
+please coordinate your maintenance window with us, so we can ensure that our team
+can provide support in a timely manner. In any event, please do not hesitate to
+:ref:`contact us <contact_us>` for assistance.
+
+
+.. _check_server_versions:
+
+Check your SecureDrop version (servers)
+---------------------------------------
+To check your SecureDrop server version, load the .onion address of your
+*Source Interface* in Tor Browser. The version number will be in the footer.
+It should currently be |version|.
+
+If you have :ref:`SSH access <verify_ssh_access>` to the servers, you can also
+check the application version from your *Admin Workstation* by running
+this command in a terminal:
+
+.. code:: sh
+
+ ssh app apt-cache policy securedrop-app-code
+
+SecureDrop servers are updated automatically with the latest release version.
+If your servers are running an old version, this indicates a major configuration
+problem, and you may need to reinstall SecureDrop. In that case, please
+:ref:`contact us <contact_us>` for assistance.
```

Comment:
I realize this is language from the existing prep guide, but on re-reading it here, it seems to me we should probably drop the "you may need to reinstall SecureDrop" here. They'll need to reinstall SecureDrop regardless of their current version in order to migrate; if their current server is so badly misconfigured that it's not even on 1.7.1, that may just imply that a careful review is required to determine if the data can be safely moved over to the new install. So I would suggest just dropping that clause, and keeping the "please contact us".